ehitchhiking-ui/src/app/shared/components/pre-loading/pre-loading.component.ts

```diff
@@ -0,0 +1,36 @@
+import {Component, Input, OnInit} from '@angular/core';
+
+@Component({
+	selector: 'app-pre-loading',
+	templateUrl: './pre-loading.component.html',
+	styleUrls: ['./pre-loading.component.sass'],
+})
+export class PreLoadingComponent implements OnInit {
+	@Input() isLoading: boolean;
+	@Input() size: string;
+
+	constructor() {}
+
+	ngOnInit() {}
+
+	defineSize(size: string) {
+		switch (size) {
```

Comment:
there is easier way to do that 
return \`gooy-${size}\`;

Reply:
done